### ISSUE TITLE
Redirect from /[lang]/login to /[lang]/me if already logged in

### ISSUE THREAD

**Issue by ldidry** [umap-project/umap#1777]
**Is your feature request related to a problem? Please describe.**
On https://framacarte.org, we added a static landing page (with Nginx, no changes in the templates). We have a button to /[lang]/login to let people log-in without going to /[lang]/ first.

We should be redirected to our maps (/[lang]/me) if we are already logged in instead of select an identity provider and sign-in.

**Describe the solution you'd like**
If already signed in, be redirected from login page to maps page.

**Comment by davidbgk** [comment]
Our default [LOGIN_REDIRECT_URL](https://docs.djangoproject.com/en/5.0/ref/settings/#std-setting-LOGIN_REDIRECT_URL) setting is currently set to `"/"` by default, it probably requires to set it to the `/me` page.

Then, we need to set the [redirect_authenticated_user](https://docs.djangoproject.com/en/5.0/topics/auth/default/#django.contrib.auth.views.LoginView.redirect_authenticated_user) parameter to `True` being aware of the potential fingerprinting issue mentioned as a Warning in that documentation. I'm not sure about the _redirect loop_ being mentioned.

**Comment by yohanboniface** [comment]
@ldidry just tried the login process in framacarte, and I've ended in my dashboard, as expected. Did you already change something ?

(Nice new landing page, btw! :) )

**Comment by ldidry** [comment]
No, we didn’t changed a thing. But were you already signed-in? If you go again to https://framacarte.org/fr/login/, are you redirected to https://framacarte.org/fr/me/ ? (I am not)

> (Nice new landing page, btw! :) )

Thanks :slightly_smiling_face:

**Comment by ldidry** [comment]
Yes, indeed :+1:

**Comment by ldidry** [comment]
Should I close the issue or is it still relevant (I prefer when /login redirects me to the application if I’m already logged-in, but I can live without that behavior).
